packages/api/src/routes/internal.ts

```diff
@@ -211,6 +214,38 @@ router.get(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /internal/cx-data/patients
+ *
+ * Returns the cx patients for a given facility used for internal scripts
+ * @param req.query.facilityId - The facility ID.
+ */
+router.get(
+  "/cx-data/patients",
```

Comment:
Any particular reason to not have this as `/internal/patient` - there's an `internal-patient.ts` file already

Reply:
Done -- moved to internal-patient.